get_radec returning coordinates of a nearby object

In my environment (astroquery v0.4.7.dev7761) Simbad.query_object('HR 8799') in get_radec returns a single object HD 218396 at 23 07 28.7157 +21 08 03.310 and test_getinfo.py fails. The coordinates written in test_get_radec() corresponds to 23 07 28.7156 +21 08 03.302, which are coordinates for one of the HR 8799 planets (presumably picked up by result_table['RA'][0] etc).

I got this, and the reason is explained above.

======================================================================= FAILURES ========================================================================
____________________________________________________________________ test_get_radec _____________________________________________________________________

    def test_get_radec():
        ra, dec = get_radec('HR8799')
        print(ra, dec)
        delta = ((ra - 346.86964833333326)**2+(dec - 21.134250555555553)**2)
>       assert delta == 0.0
E       assert 5.111882798134094e-12 == 0.0

tests/utils/test_getinfo.py:9: AssertionError
----------------------------------------------------------------- Captured stdout call ------------------------------------------------------------------
346.86964875 21.134252777777778
================================================================ short test summary info ================================================================
FAILED tests/utils/test_getinfo.py::test_get_radec - assert 5.111882798134094e-12 == 0.0
============================================================= 1 failed, 8 passed in 29.18s ==============================================================

--- a/tests/utils/test_getinfo.py
+++ b/tests/utils/test_getinfo.py
@@ -3,10 +3,10 @@ from pyird.utils.getinfo import get_radec
 
 
 def test_get_radec():
-    ra, dec = get_radec('HR8799')
+    ra, dec = get_radec('HR8799 A')
     print(ra, dec)
-    delta = ((ra - 346.86964833333326)**2+(dec - 21.134250555555553)**2)
-    assert delta == 0.0
+    delta = ((ra - 346.86964875)**2+(dec - 21.134252777777778)**2)
+    assert pytest.approx(delta) == 0.0
 
 
 if __name__ == '__main__':

